cosmolog/cosmologger.py

```diff
@@ -180,11 +181,11 @@ def _validate_payload(cls, payload):
 
     @classmethod
     def _validate_payload_key(cls, key):
-        m = re.match(r'[a-zA-Z0-9][a-zA-Z0-9_]+', key)
+        m = re.match(r'[a-zA-Z0-9][a-zA-Z0-9_\-\.]+', key)
```

Comment:
Add leading underscores while we're at it?

Reply:
Hm. That's a good question. There is something a bit odd about keys with leading underscores, dashes, or dots. I vote "no until we need it". LMK if you think otherwise.